core/api/src/main/java/com/blazebit/persistence/PaginatedCriteriaBuilder.java

```diff
@@ -54,6 +62,40 @@
      */
     public boolean isKeysetExtraction();
 
+    /**
+     * Enable or disables executing the count query which influences whether {@link PagedList#getTotalSize()} is available.
```

Comment:
Enables or disables the execution of the count query which determines whether {@link PagedList#getTotalSize()} is available.